cli/command.go

```diff
@@ -122,14 +140,19 @@ func execute() {
 
 	/* Everything's good so far, ready to start */
 	glog.Infoln("Starting freenas-provisioner with the following parameters:")
-	glog.Infof("  Freenas address: https://%s:%d\n", *host, *port)
+	glog.Infof("  Freenas address: %s://%s:%d\n", *protocol, *host, *port)
 	glog.Infof("  Insecure: %t\n", *insecure)
 	glog.Infof("  pool: %s\n", filepath.Join(*mountpoint, *pool))
 	glog.Infof("  parentDataset: %s\n", *parentDataset)
 
-	// Create an InClusterConfig and use it to create a client for the controller
-	// to use to communicate with Kubernetes
-	config, err := rest.InClusterConfig()
+	if *kubeconfig != "" {
```

Comment:
I presume `kubeconfig` var can be nil (no default value set)

Reply:
@nmaupu yeah, if it's not present OR empty the provisioner attempts to use in-cluster config.